.changeset/loud-rules-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rspress/theme-default": major
```

Comment:
maybe a minor change is better

Reply:
Thank you for pointing out the error in this and helping me to fix it, it has been fixed